src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/combination_listing.html.twig

```diff
@@ -52,29 +52,57 @@
       {{ form_widget(form._token) }}
     {% endif %}
     <thead class="thead-default">
-      <tr class="column-headers">
-        <th scope="col" id="id-header">
-          <div class="md-checkbox md-checkbox-inline">
-            <label>
-              <input type="checkbox" id="bulk-select-all-in-page">
+    <tr class="column-headers">
+      <th scope="col" id="id-header">
+        <div class="dropdown" id="bulk-all-selection-dropdown">
+          <button type="button"
+                  class="md-checkbox dropdown-toggle"
+                  data-toggle="dropdown"
+                  aria-haspopup="true"
+                  aria-expanded="false"
+          >
+            <label for="bulk-all-preview" data-label="(%combinations_number%)"><span></span></label>
```

Comment:
```suggestion
            <label for="bulk-all-preview" data-label="(%combinations_number% / %total_combinations%)"><span></span></label>
```

Reply:
I'm wondering with this dropdown, is it possible to click on the checkbox directly without opening the dropdown? Or does a click on the checkbox opens the dropdown and then we need to click on one of the two others?

Reply:
yep we mus click on one of dropdown checkboxes, the preview one is not selectable by user (when you click it opens up a dropdown)